### PR TITLE
Todo-List Application 🤖 

### PR DESCRIPTION
**Pull request by evansnyamekye**
List-Structure 🦺: 

**WEBPACK WORK FLOW INTEGRATION** 
As required by this milestone, I used JavaScript code to built this simple todo-list App📴.
I used HTML to build up the entire html code structure of todo App.
I configured linters, eslintrc and any other dependencies to handle the workflow of my codebase 🧥.
I used modules feature to hold the Data file and it functionalities.
I used CSS to style the layout of the website through the webpack workflows deployment .

Thank you 😀

### CODE REVIEW

**Review by Zuhaib042 — CHANGES_REQUESTED** [review]
Hi @evansnyamekye 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights
✅ Added webpack to the project
✅ The css is getting loaded with the help of webpack

## Required Changes ♻️
- [ ] The package.json and package-lock.json files are missing so I cannot execute any npm command. kindly consider adding those two files in this feature branch so that the reviewer can check if distribution files are generated by webpack and served by webpack dev server from /dist folder.
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by evansnyamekye** [comment]
**@Zuhaib042** 
Thank you for the feedback. 
Both package.json and package-lock.json has been added. 

Kindly check 👍.

File: README.md [hunk 1]
```diff
@@ -67,7 +65,7 @@ create a new task, mark the task as completed  and delete them from local storag
 <!-- Live Demo -->
 
 ## 🚀 Live Demo <a name="live-demo"></a>
-<a href="">Live-Demo</a>
+
```
Comment (shahadat3669) [inline README.md:68]: - [ ] [Optional] Well!👍 Considering that your project doesn't have a live version, it's advisable to remove the **Live Demo** section and the corresponding entry from the top list. This step will enhance the clarity and presentation of your project. Keep up the good work! 🚀

**Review by shahadat3669 — APPROVED** [review]
Hi @evansnyamekye, 

Your project is complete!  There is nothing else to say other than it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights
✅ Webpack is used to set up the project.
✅ Has an array of simple *to do tasks*.
✅ Has a function that iterates over the tasks array and populates the HTML.


## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

## Have any doubt ❓

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.

<table  width="100%" align="center">
  <tr>
    <td colspan="4" align="center">
      <strong>Contact me: Shahadat Hossain</strong>
    </td>
  </tr>
  <tr>
    <td align="center">
      <a href="https://github.com/shahadata3669">
        GitHub
      </a>
    </td>
    <td align="center">
      <a href="https://www.linkedin.com/in/shahadat-cseng/">
        LinkedIn
      </a>
    </td>
    <td align="center">
      <a href="https://app.slack.com/client/T47CT8XPG/D05GT4E4WAY/rimeto_profile/U04426SD71V">
        Slack
      </a>
    </td>
    <td align="center">
      <a href="mailto:shahadat3669@gmail.com?subject=Microverse%20Code%20Review">
        Email
      </a>
    </td>
  </tr>
</table>

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**